R/rdply.r

```diff
@@ -31,8 +30,12 @@ rdply <- function(.n, .expr, .progress = "none", .id = NA) {
   res <- .rlply_worker(.n, .progress,
                        eval.parent(substitute(function() .expr)))
   names(res) <- seq_len(.n)
-  labels <- data.frame(.n = seq_len(.n))
-  if (!is.na(.id))
-    names(labels) <- .id
+  if (is.null(.id))
```

Comment:
Can you please wrap all blocks in `{}` with appropriate spacing. Full style guide at http://r-pkgs.had.co.nz/r.html#style